syncd/syncd_saiswitch.cpp

```diff
@@ -599,9 +589,7 @@ void SaiSwitch::collectCounters(
             const std::string &field = sai_serialize_port_stat(m_supported_counters[idx]);
             const std::string &value = std::to_string(counters[idx]);
 
-            swss::FieldValueTuple fvt(field, value);
-
-            values.push_back(fvt);
+            values.push_back( swss::FieldValueTuple(field, value) );
```

Comment:
dont gibe extra spaces '( swss' this formatting is not used anywhere in this project